timesketch/lib/analyzers/sequence_sessionizer_test.py

```diff
@@ -0,0 +1,430 @@
+                MockDataStore)
+    def test_sessionizer(self):
+        """Test basic sequence sessionizer functionality."""
+        index = 'test_index'
+        sketch_id = 1
+        sessionizer = ManyEventsSequenceSessionizer(index, sketch_id)
+        self.assertIsInstance(sessionizer, SequenceSessionizerSketchPlugin)
+        self.assertEqual(index, sessionizer.index_name)
+        self.assertEqual(sketch_id, sessionizer.sketch.id)
+
+    @mock.patch('timesketch.lib.analyzers.interface.ElasticsearchDataStore',
+                MockDataStore)
+    def test_one_session(self):
+        """Test one sequence of events is finded and allocated as a session."""
+        with mock.patch.object(SequenceSessionizerSketchPlugin,
+                               'event_stream',
+                               return_value=_create_mock_event(
+                                   0, 2, one_session_args, time_diffs=[1])):
+            index = 'test_index'
+            sketch_id = 1
+            sessionizer = ManyEventsSequenceSessionizer(index, sketch_id)
+            message = sessionizer.run()
+            self.assertEqual(
+                message,
+                'Sessionizing completed, number of {0:s} session created: 1'.
+                format(sessionizer.session_type))
+
+            ds = MockDataStore('test', 0)
+            # Events that are not part of the sequence but are between
+            # significant events from the event sequence considered as a session
+            # are part of the significant events' session.
+            for i in range(0, 101):
+                event = ds.get_event('test_index', str(i), stored_events=True)
+                self.assertEqual(event['_source'][sessionizer.session_type], 1)
+
+    @mock.patch('timesketch.lib.analyzers.interface.ElasticsearchDataStore',
+                MockDataStore)
+    def test_multiple_sessions(self):
+        """Test multiple sessions are finded and allocated correctly."""
+        with mock.patch.object(SequenceSessionizerSketchPlugin,
+                               'event_stream',
+                               return_value=_create_mock_event(
+                                   0,
+                                   4,
+                                   two_sessions_args,
+                                   time_diffs=[1, 1, 1])):
+            index = 'test_index'
+            sketch_id = 1
+            sessionizer = ManyEventsSequenceSessionizer(index, sketch_id)
+            message = sessionizer.run()
+            self.assertEqual(
+                message,
+                'Sessionizing completed, number of {0:s} session created: 2'.
+                format(sessionizer.session_type))
+
+            ds = MockDataStore('test', 0)
+            for i in range(0, 100):
+                event = ds.get_event('test_index', str(i), stored_events=True)
+                self.assertEqual(event['_source'][sessionizer.session_type], 1)
```

Comment:
In tests, I would rather be explicit here and use the actual hardcoded string instead of the variable `sessionizer.session_type` when possible.